ui/packages/catalog-admin-module-sources/src/main/webapp/js/model/Source.js

```diff
@@ -33,18 +33,28 @@ define([
         Source.Model = Backbone.Model.extend({
             configUrl: "../jolokia/exec/org.codice.ddf.ui.admin.api.ConfigurationAdmin:service=ui",
             idAttribute: 'name',
-            defaults: function() {
-              return {
-                currentUrl:     undefined,
-                isLoopbackUrl:   undefined,
-              };
+            defaults: {
+                currentUrl: undefined,
+                isLoopbackUrl: undefined,
             },
             initialize: function () {
                 this.set('currentConfiguration', undefined);
                 this.set('disabledConfigurations', new Source.ConfigurationList());
                 this.listenTo(this, 'change:currentConfiguration', this.updateCurrentBinding);
+                this.listenTo(this, 'remove', this.onRemove);
```

Comment:
I'm not as familiar with this code.  What is the remove event here?  This looks like a model to me, not a collection, so I'm a slightly confused.

Reply:
The 'remove` event gets triggered when calling `collection.remove()`. See http://backbonejs.org/#Collection-remove